services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: datetime = None
+                 ) -> Order:

```

Comment:
Place each argument on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability